src/pages/messages/components/TicketListItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { RegistrationListItem } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { Ticket, PublishingTicket } from '../../../types/publication_types/messages.types';
+import { Registration, emptyRegistration } from '../../../types/registration.types';
+import { PublishingRequestMessagesColumn } from './PublishingRequestMessagesColumn';
+
+const ticketColor = {
+  PublishingRequest: 'publishingRequest.main',
+  DoiRequest: 'doiRequest.main',
+  GeneralSupportCase: 'generalSupportCase.main',
+};
+
+interface TicketListItemProps {
+  ticket: Ticket;
+}
+
+export const TicketListItem = ({ ticket }: TicketListItemProps) => {
+  const { t } = useTranslation();
+
+  const { id, identifier, mainTitle, contributors, publicationInstance } = ticket.publication;
+  const registration: Registration = {
+    ...emptyRegistration,
+    identifier,
+    id,
+    entityDescription: emptyRegistration.entityDescription
+      ? {
+          ...emptyRegistration.entityDescription,
+          mainTitle,
+          contributors,
+        }
+      : undefined,
+  };
+  if (registration.entityDescription?.reference?.publicationInstance) {
+    registration.entityDescription.reference.publicationInstance.type = publicationInstance.type;
+  }
+
+  const msAge = new Date().getTime() - new Date(ticket.modifiedDate).getTime();
+  const daysAge = Math.ceil(msAge / (1000 * 3600 * 24));
```

Comment:
kan hardkode 86_400_000 for å unngå to unødvendige multiplikasjonsoperasjoner :))